packages/page-staking/src/react-components/ProducerPerformance/index.tsx

```diff
@@ -27,7 +27,17 @@ function queryAddress (address: string) {
   window.location.hash = `/staking/query/${address}`;
 }
 
-function Address ({ address, blocksCreated, filterName, rewardPercentage, session }: Props): React.ReactElement<Props> | null {
+/**
+ * A reusable component describing block production performance of a block production committee member.
+ * @param address Validator's account id
+ * @param blocksCreated How many blocks the validator created in a session; is optional only for better UX experience,
+ *                      ie when empty, it means parent component is still calculating data
+ * @param filterName a pattern which is used to filter validator, either by account id, domain or identity; can be empty
+ * @param rewardPercentage a percent as string, e.g. '100.0%'
+ * @param session session number, optional. If specified, additional column is rendered.
+ * @constructor
+ */
+function ProducerPerformance ({ address, blocksCreated, filterName, rewardPercentage, session }: Props): React.ReactElement<Props> | null {
```

Comment:
I think we don't need the `@constructor`

Reply:
My IDE proposed it but indeed it's not needed, removed.